estate/models/estate_property.py

```diff
@@ -0,0 +1,21 @@
+from odoo import fields, models
+
+class RecurringPlan(models.Model):
+    _name = "estate.property"
+    _description = "estate property"
+    
+    name = fields.Char(required = True)
```

Comment:
```suggestion
    name = fields.Char(required=True)
```
don't add unnecessary spaces. do this for all fields